### PR TITLE
Update query to work with Solr 7.2

### PR DESCRIPTION
**Pull request by jcoyne**
See the first item in:
https://lucene.apache.org/solr/7_2_0/changes/Changes.html#v7.2.0.upgrade_notes

This probably needs to be backported.

### CODE REVIEW

**Comment by jcoyne** [comment]
It's still failing the did_you_mean_spec and search_spec which first started failing in https://travis-ci.org/projectblacklight/blacklight/builds/319183477

**Comment by jcoyne** [comment]
We should probably merge #1821 until we figure out what else needs fixing here.

File: spec/models/blacklight/solr/search_builder_spec.rb [hunk 1]
```diff
@@ -240,6 +240,14 @@
     describe "solr parameters for a field search from config (subject)" do
       let(:user_params) { subject_search_params }
 
+      before do
+        # pre solr-7.2-style configuration with local params
```
Comment (jcoyne) [inline spec/models/blacklight/solr/search_builder_spec.rb]: I'm not sure what this comment is telling me.  Is it "You can set..."?

File: spec/features/search_filters_spec.rb [hunk 2]
```diff
@@ -187,16 +187,18 @@
     visit root_path
     within(".blacklight-subject_topic_facet") do
       expect(page).not_to have_selector(".card-block", :visible => true)
-      click_link "Topic"
+      find(".card-header").click
```
Comment (jcoyne) [inline spec/features/search_filters_spec.rb]: How is Solr 7.2 affecting the front end here?

Comment (cbeer) [reply spec/features/search_filters_spec.rb]: No idea, but I'm getting test failures on the branch without it. Testing in the browser works fine.

File: spec/models/blacklight/solr/search_builder_spec.rb (hunk 1 above)
Comment (cbeer) [reply spec/models/blacklight/solr/search_builder_spec.rb]: The tests in this block expect the solr local params, which don't work at all in Solr 7.2, so we have to set that configuration in this unit test to test the feature.

**Pull request merged**